src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/ActiveDirectoryAuthenticationProvider.cs

```diff
@@ -198,19 +201,23 @@ public override void BeforeUnload(SqlAuthenticationMethod authentication)
                     try
                     {
                         result = await app.AcquireTokenSilent(scopes, account).ExecuteAsync();
+                        SqlClientEventSource.Log.TryTraceEvent("AcquireTokenAsync | Acquired access token for {0} auth mode. Expiry Time: {1}", parameters.AuthenticationMethod, result.ExpiresOn);
```

Comment:
Can we add extra information to distinguish these three identical event trace messages (Line: 204, 209, and 215)?